sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1820,8 +1990,28 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public static bool operator !=(Azure.ResourceManager.Avs.Models.AzureHybridBenefitType left, Azure.ResourceManager.Avs.Models.AzureHybridBenefitType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CloudLinkProvisioningState : System.IEquatable<Azure.ResourceManager.Avs.Models.CloudLinkProvisioningState>
```

Comment:
can we rename this to `AvsCloudLinkProvisioningState` by adding a
```
CloudLinkProvisioningState: AvsCloudLinkProvisioningState
```
in the `rename-mapping` section in your autorest.md? please run the `dotnet build /t:GenerateCode` to regenerate to make it take effect.